tools/scripts/gitrelease.pl

```diff
@@ -1895,46 +2306,54 @@ sub remedy_gen_doxygen {
     return 1;
   }
 
-  my $curdir = getcwd;
-  $ENV{DDS_ROOT} = $settings->{clone_dir};
+  my $dox_dir = $settings->{doxygen_dir};
+  mkdir($dox_dir) if (!-d $dox_dir);
+  $ENV{DDS_ROOT} = $settings->{tgz_worktree};
   $ENV{ACE_ROOT} = $settings->{ace_root};
-  chdir($ENV{DDS_ROOT});
-  my $result = run_command("$ENV{DDS_ROOT}/tools/scripts/generate_combined_doxygen.pl . -is_release");
-  chdir($curdir);
-  return $result;
+  run_command(["./tools/scripts/generate_combined_doxygen.pl", $dox_dir, '-is_release'],
+    chdir => $ENV{DDS_ROOT}, autodie => 1);
+
+  print "Converting doxygen files to Windows line endings...\n";
```

Comment:
Probably not, I was just trying to keep what it was doing. If you don't think it's necessary we can remove it.

Reply:
Doesn't have to be in this PR, but we can eventually go to a single Doxygen artifact to save time & space

Reply:
It has only generated a single artifact since #3574. Before it was making a tar with unix line endings and a zip with windows line endings. I removed the tar in that PR, but the zip kept the windows line endings.

Reply:
Yeah, I misremembered what was happening before this PR.  Unlike the source code artifacts it may make sense for it to be a .zip (everybody can extract) that contains LF line endings.  Perhaps that would be confusing, but it also may be odd for a Linux user to encounter CR-LF line endings.